misk-redis-testing/src/test/kotlin/misk/redis/AbstractRedisTest.kt

```diff
@@ -356,9 +284,7 @@ class FakeRedisTest {
     redis[key] = "Not a number".encodeUtf8()
 
     // Exercise
-    assertThrows<IllegalArgumentException> {
-      redis.incrBy(key, 3)
-    }
+    assertThrows<RuntimeException> { redis.incrBy(key, 3) }
```

Comment:
The RealRedis client actually throws a JedisDataException here. Both are RuntimeExceptions, hence this change.

I'm unsure if misk RealRedis should remap Jedis specific exceptions to something more Kotlin-y, or if the Fake should throw something other than an IllegalArgumentException, but I will not change that behaviour in this PR.

Reply:
yeah, I would change that in a different PR. 

My thoughts there:
* FakeRedis should imitate RealRedis as close as possible. 
* Should we remap Jedis specific exceptions? I think it depends on how much you wanna hide the usage of jedis in this library. I wonder how consistent we're in other part of misk where we use other libraries. In my mind the tradeoff are:
  * [+]  The abstractions encapsulates redis concern better because you don't force misk users to know how both use misk-redis and jedis. 
  * [-] it can become a shallow abstractions where we're just mapping things from one side to another. 

I wish we could define some misk principles on how to wrap other libraries.

Reply:
> I wish we could define some misk principles on how to wrap other libraries.

That would be amazing. Is misk meant to be a thin kotlinized wrapper around things? Is it meant to provide its own set of opinioned abstractions? Presently... It does both, a lot.